spec/index.html

```diff
@@ -299,6 +299,66 @@ <h3>Object Lists</h3>
     Turtle provides a number of ways of writing each.
   </p>
 
+  <section id="sec-version">
+    <h3>Version Announcement</h3>
+
+    <p>The Turtle language has evolved since its origin, and RDF 1.2
+      introduces new features which may confuse non-compliant parsers.
+      RDF 1.2 Turtle introduces the
+      <a href="#grammar-production-sparqlVersion"><code>VERSION</code></a> and
+      <a href="#grammar-production-version"><code>@version</code></a> directives
+      along with an optional `version` <a href="#sec-mediaReg">HTTP Media Type parameter</a>.
+      When serializing Turtle with new features such as base direction or triple terms,
+      authors can announce the presense of these features using these directives.
+    </p>
+
+    <pre id="ex-version-decl"
+         class="example turtle" data-transform="updateExample"
+         title="Version announcement">
+      <!--
+      @version "1.2" .
+      @prefix : <http://example.com/> .
+      :a :name "Alice" ~ :t {| :statedBy :bob ; :recorded "2021-07-07"^^xsd:date |} .
+      -->
+    </pre>
+
+    <p>Alternatively, the SPARQL style directive can also be used
+      (note that there is no trailing <a href="#cp-full-stop"><code title="full stop">.</code></a>):</p>
+
+    <pre id="ex-version-decl"
+         class="example turtle" data-transform="updateExample"
+         title="SPARQL-style version announcement">
+      <!--
+      VERSION "1.2"
+      PREFIX : <http://example.com/>
+      :a :name "Alice" ~ :t {| :statedBy :bob ; :recorded "2021-07-07"^^xsd:date |} .
+      -->
+    </pre>
+
+    <p>When providing content over HTTP, serveres can announce the version
+      using the optional `version` Content-Type parameter:</p>
+
+    <pre id="ex-version-decl"
+         class="example turtle" data-transform="updateExample"
+         title="HTTP version announcement">
+      <!--
+      GET /document.ttl HTTP/1.1
+      Host: example.com
+      Accept: text/turtle; version=1.2
+      -->
+    </pre>
+
+    <p>In the absense of an explicit version announcement,
+      processors should treat the document as having version "1.2".</p>
```

Comment:
Shouldn't the "unversioned" be treated as 1.0/1.1 (which are handled the same), and only treat the version as 1.2 when declared as such?
```suggestion
    <p>In the absense of an explicit version announcement,
      processors should treat the document as having version "1.1".</p>
```

Reply:
I don't think this is something we've agreed on. By default, Turtle 1.2 parsers should be able to parse all the 1.2 features. The directive exists so that pre-1.2 parsers can detect early that the document uses features they won't be able to parse.

Reply:
The lack of a version declaration is what is found in all existing data, which is RDF 1.0 or 1.1, and so it should be treated.

The *presence* of a version declaration begins with RDF 1.2, but I think (hope) it should cause no harm if `version 1.1` is added to existing data.

I think pre-1.2 parsers will need to be updated to react appropriately when encountering a version 1.2 declaration, but will need no update to appropriately handle undeclared version 1.1. I don't know what these parsers will do if they encounter a version 1.1 declaration.

Reply:
Any existing parsers that follow the 1.1 grammar will fail with a version declaration, other than in Content-Type. It would be silly to update a 1.1 parser part way to 1.2 just to parse `@version` or `VERSION` and not the other constructs. I think we need to discuss this further with the group.

Reply:
I also remember the sentiment in our last call being that explicitly mentioning version 1.2 should be optional (SHOULD), even if new 1.2 features are used.